plugins/kubernetes-backend/src/service/KubernetesFetcher.test.ts

```diff
@@ -812,6 +812,129 @@ describe('KubernetesFetcher', () => {
         const [[{ agent }]] = httpsRequest.mock.calls;
         expect(agent.options.rejectUnauthorized).toBe(false);
       });
+
+      it('should use a x509 client cert authentication strategy to consume kubeapi when backstage-kubernetes-auth field is not provided and the authStrategy enables x509 client cert authentication - fetchObjectsForService', async () => {
```

Comment:
missing [this suggestion](https://github.com/backstage/backstage/pull/21842#discussion_r1440978611)

Reply:
Done, sorry I forgot to update my local branch :)